tests/tests_integration/test_api/test_simulators/test_routines.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+
+from cognite.client._cognite_client import CogniteClient
+from cognite.client.data_classes.simulators.filters import CreatedTimeSort, SimulatorRoutinesFilter
+from cognite.client.data_classes.simulators.routines import SimulatorRoutineWrite
+from cognite.client.utils._text import random_string
+
+
+@pytest.mark.usefixtures(
+    "seed_resource_names",
+    "seed_simulator_routines",
+)
+class TestSimulatorRoutines:
+    def test_list_routines(self, cognite_client: CogniteClient, seed_resource_names) -> None:
+        model_unique_external_id = seed_resource_names["simulator_model_external_id"]
+
+        routines = cognite_client.simulators.routines.list(
+            filter=SimulatorRoutinesFilter(model_external_ids=[model_unique_external_id])
+        )
+
+        assert len(routines) > 0
+
+    def test_create_routine(self, cognite_client: CogniteClient, seed_resource_names) -> None:
+        routine_external_id_new = random_string(10)
+
+        routine_to_create = SimulatorRoutineWrite(
+            name="sdk-test-routine",
+            model_external_id=seed_resource_names["simulator_model_external_id"],
+            simulator_integration_external_id=seed_resource_names["simulator_integration_external_id"],
+            external_id=routine_external_id_new,
+        )
+
+        routine_created = cognite_client.simulators.routines.create(routine_to_create)
+        assert routine_created is not None
+        assert routine_created.external_id == routine_external_id_new
+        cognite_client.simulators.routines.delete(external_ids=[routine_external_id_new])
+
+    def test_sort(self, cognite_client: CogniteClient, seed_resource_names) -> None:
+        simulator_integration_unique_external_id = seed_resource_names["simulator_integration_external_id"]
+        routine_external_ids = [random_string(10) for _ in range(3)]
+
+        for routine_external_id in routine_external_ids:
+            routine_to_create = SimulatorRoutineWrite(
+                name="sdk-test-routine",
+                model_external_id=seed_resource_names["simulator_model_external_id"],
+                simulator_integration_external_id=simulator_integration_unique_external_id,
+                external_id=routine_external_id,
+            )
+
+            cognite_client.simulators.routines.create(routine_to_create)
```

Comment:
Move these out as a fixture, remove the randomness, and do not delete them. Instead lets have a permanent list of routines used in the test.